README.md

```diff
@@ -20,7 +20,7 @@ SHAKE is an extendable-output functions (XOFs), i.e. it can produce a variable l
 The default output character set is hex digits: `a-z0-9`
 
 See also:
-- https://github.com/cds-snc/url-shortener/issues/87
+- https://github.com/cds-snc/URL-shortener/issues/87
```

Comment:
Don't think this'll need to be capitalized? 🤔

Reply:
I agree - it shouldn't be.  I'll get it fixed!